README.md

```diff
@@ -1,6 +1,15 @@
 # Prewired Website
 
-Code for http://prewired.org.
+Code for [http://prewired.org](http://prewired.org).
```

Comment:
I know that we have an https redirect, but while I'm looking at this, I wonder if we should be linking to https://prewired.org here?

Reply:
Yes, this would be better:
Code for [https://prewired.org](https://prewired.org).